ts/Core/Palette.ts

```diff
@@ -0,0 +1,88 @@
+/**
+ * Palette for Highcharts. Do not edit this file! This file is generated using
+ * the 'gulp palette' task. Palette colors are defined in highcharts.scss.
+ */
+/* eslint comma-dangle: 0, max-len: 0 */
+import type ColorString from './Color/ColorString';
+const palette = {
+    /**
+     * Colors for data series and points.
+     */
+    colors: [
+        '#7cb5ec',
+        '#434348',
+        '#90ed7d',
+        '#f7a35c',
+        '#8085e9',
+        '#f15c80',
+        '#e4d354',
+        '#2b908f',
+        '#f45b5b',
+        '#91e8e1'],
+    /**
+     * Chart background, point stroke for markers and columns etc
+     */
+    backgroundColor: '#ffffff' as ColorString,
+    /**
+     * Strong text.
+     */
+    neutralColor100: '#000000' as ColorString,
+    /**
+     * Main text and some strokes.
+     */
+    neutralColor80: '#333333' as ColorString,
+    /**
+     * Axis labels, axis title, connector fallback.
+     */
+    neutralColor60: '#666666' as ColorString,
+    /**
+     * Credits text, export menu stroke.
+     */
+    neutralColor40: '#999999' as ColorString,
+    /**
+     * Disabled texts, button strokes, crosshair etc.
+     */
+    neutralColor20: '#cccccc' as ColorString,
+    /**
+     * Grid lines etc.
+     */
+    neutralColor10: '#e6e6e6' as ColorString,
+    /**
+     * Minor grid lines etc.
+     */
+    neutralColor5: '#f2f2f2' as ColorString,
+    /**
+     * Tooltip backgroud, button fills, map null points.
+     */
+    neutralColor3: '#f7f7f7' as ColorString,
+    /**
+     * Drilldown clickable labels, color axis max color.
+     */
+    highlightColor100: '#003399' as ColorString,
+    /**
+     * Selection marker, menu hover, button hover, chart border, navigator series.
+     */
+    highlightColor80: '#335cad' as ColorString,
+    /**
+     * Navigator mask fill.
+     */
+    highlightColor60: '#6685c2' as ColorString,
+    /**
+     * Ticks and axis line.
+     */
+    highlightColor20: '#ccd6eb' as ColorString,
+    /**
+     * Pressed button, color axis min color.
+     */
+    highlightColor10: '#e6ebf5' as ColorString,
+    /**
+     * Positive indicator color
+     */
+    indicatorPositiveLine: '#06b535' as ColorString,
+    /**
+     * Negative indicator color
+     */
+    indicatorNegativeLine: '#f21313' as ColorString,
+
+};
```

Comment:
If you write this as a namespace with a bunch of constants, you avoid forced casting and the JS outcome would be the same, plus you get a uppercase namespace object.
```ts
namespace Palette {
    …
    /**
     * Negative indicator color
     */
    export const indicatorNegativeLine: ColorString = '#f21313';
}
export default Palette;
```

Reply:
Sorry, I forgot that we need a JSON-like structure.